Documentation/ApiOverview/JavaScript/Navigation/Index.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../../../Includes.txt
+
+
+.. _javascript-navigation:
+
+=========================
+Navigation via JavaScript
+=========================
+
+Navigate to URL
+===============
+
+Navigates to URL once selected drop-down was changed:
+
+.. code-block:: html
+
+   <select data-global-event="change" data-action-navigate="$value">'
+
+`$value` refers to selected value.
+
+Navigate to URL with Data
+=========================
+
+Navigate to an URL on change of a drop-down, including the selected value in the URL:
```

Comment:
```suggestion
Navigate to a URL on change of a drop-down, including the selected value in the URL:
```
Should this be similar to the above wording "Navigate to a URL once selected drop-down is changed, and include the selected value in the URL" ?